$NetBSD: patch-runtime_mono-wrapper.in,v 1.1 2019/08/25 16:37:01 maya Exp $

Mark the mono executable as not MPROTECT safe during the build.

--- runtime/mono-wrapper.in.orig	2019-07-18 07:46:08.000000000 +0000
+++ runtime/mono-wrapper.in
@@ -8,4 +8,5 @@ then
 fi
 MONO_EXECUTABLE=${MONO_EXECUTABLE:-"$r/@mono_runtime@"}
 export MONO_CFG_DIR PATH
+paxctl +m $(readlink -f "$MONO_EXECUTABLE") | true 2>/dev/null
 exec "$r/libtool" --mode=execute "${MONO_EXECUTABLE_WRAPPER}" --config "@mono_cfg_dir@/mono/config" "$@"
